src/App.js

```diff
@@ -1,24 +1,36 @@
-import logo from './logo.svg';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import './App.css';
+import Header from './components/Header/Header';
+import Home from './views/Home/Home';
+import TeamDetails from './views/Teams/TeamDetails';
+import Teams from './views/Teams/Teams';
+import Players from './views/Players/Players';
+import PlayerDetails from './views/Players/PlayerDetails';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Header />
+      <div className="App">
+        <Switch>
+          <Route exact path='/'>
+            <Home />
+          </Route>
+          <Route exact path='/teams/:id'>
+            <TeamDetails />
+          </Route>
+          <Route path='/teams'>
+            <Teams />
+          </Route>
+          <Route path='/players/:id'>
+            <PlayerDetails />
+          </Route>
+          <Route path='/players'>
```

Comment:
Nice job having the more specific routes above the generic routes so our navigation is not mismatched.